dinov2/hub/dinotxt.py

```diff
@@ -62,16 +62,21 @@ def dinov2_vitl14_reg4_dinotxt_tet1280d20h24l():
     return model
 
 
-def get_tokenizer():
+def get_tokenizer(local_path=None):
     from .text.tokenizer import Tokenizer
     import requests
     from io import BytesIO
 
     url = _DINOV2_BASE_URL + "/thirdparty/bpe_simple_vocab_16e6.txt.gz"
     try:
-        response = requests.get(url)
-        response.raise_for_status()
-        file_buf = BytesIO(response.content)
+        if not local_path:
+            response = requests.get(url)
+            response.raise_for_status()
+            content = response.content
```

Comment:
@cijose Do you remember why `requests` was pulled as dependency? Couldn't we simply use builtin Python modules instead?

```
    with urllib.request.urlopen(url) as f:
        content = f.read()
```